features/admin.claims.v1/components/edit/local-claim/edit-basic-details-local-claims.tsx

```diff
@@ -964,8 +964,8 @@ export const EditBasicDetailsLocalClaims: FunctionComponent<EditBasicDetailsLoca
                         required={ false }
                         defaultValue={ claim?.multiValued }
                         data-testid={ `${testId}-form-multi-valued-input` }
-                        hint={ t("claims:local.forms.multiValuedHint") }
-                        readOnly={ isReadOnly }
+                        hint={ !isSystemClaim && t("claims:local.forms.multiValuedHint") }
```

Comment:
We do not hide the hint in read-only mode in other fields. Shall we consider showing a different hint (to communicate why the field is disabled) for system claims rather than hiding it?

Reply:
This is fixed, and updated the screenshots.